### PR TITLE
committing to LEASE-123 instead of main

### PR DESCRIPTION
**Pull request by naomiruele**
@yuvavts could you take a look at this and review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,10 @@
 <body>
     <h3>Today we are going to refresh all we've learnt in Git so far</h3>
     <h4>Hope your day goes good</h4>
+    <h5>I am trying to push to a branch that is not main</h5>
     <p>Adding a new line here</p>
     <p>This will be the last line</p>
+
```
Comment (yuvavts) [inline index.html:15]: can you remove this empty line.

adding this comment to show you how engineers can request for more changes by commenting and not approving.

**Review by yuvavts — CHANGES_REQUESTED** [review]
requesting changes

File: index.html (hunk 1 above)
Comment (naomiruele) [reply index.html:15]: @yuvavts I have made the change!

**Review by yuvavts — APPROVED** [review]
Looks good

**Pull request merged**